packages/engine-multi/src/api/execute.ts

```diff
@@ -82,7 +84,9 @@ const execute = async (context: ExecutionContext) => {
         threadId: '-', // no thread at this point
         message: {
           level: 'debug',
-          message: [`Timeout: ${workerOptions.timeout / 1000}s`],
+          message: JSON.stringify([
```

Comment:
Again, I should just be able to send an array with a string here. No need to make my life difficult.